openapi.yml

```diff
@@ -127,7 +127,7 @@ paths:
         - persons
       summary: Returns metadata of images associated with a person ordered by newest to oldest date.
       parameters:
-        - $ref: "#/components/parameters/HmppsId"
+        - $ref: "#/components/parameters/hmppsId"
         - $ref: "#/components/parameters/Page"
```

Comment:
Nice formalisation, my question is are we going to commit to all parameters being lowercase i.e Page and PerPage?

Reply:
we probs should too